lib/qless/server/views/layout.erb

```diff
@@ -268,6 +268,11 @@
       });
     }
 
+    var view_and_move_to_pt = function(jid, story_id) {
+      window.open('https://www.pivotaltracker.com/story/show/<%=story_id%>', '_blank')
+      job.untag(jid, 'auto-gen-pt')
```

Comment:
There's no `job` variable in this scope. I think what you want is `untag(jid, 'auto-gen-pt');`